assignments/callbacks.js

```diff
@@ -52,4 +62,5 @@ function removeDuplicates(array, cb) {
   // removeDuplicates removes all duplicate values from the given array.
   // Pass the duplicate free array to the callback function.
   // Do not mutate the original array.
+    return cb(removeDuplicates());
```

Comment:
A way you could do this stretch is using the `Set` object that ES6 provides
https://www.geeksforgeeks.org/sets-in-javascript/
```
const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum', 'yo-yo'];

function removeDuplicates(arr, cb) {  
  cb(arr);
}

function unique(arr) {
  // Set stores unique values of an array 
  let unique = [...new Set(arr)]; 
  console.log(unique) 
}

removeDuplicates(items, unique );
```